galaxy_ng/tests/integration/api/test_artifact_upload.py

```diff
@@ -6,9 +6,10 @@
 
 import pytest
 from orionutils.generator import build_collection, randstr
+from pkg_resources import parse_version
 
 from galaxy_ng.tests.integration.constants import USERNAME_PUBLISHER
-from ..conftest import is_hub_4_5
+from ..conftest import is_hub_4_5, get_hub_version
```

Comment:
I know this line was already here, but I think we probably shouldn't import from conftest. Stuff in there should be fixtures that are "magically" imported via pytest. https://stackoverflow.com/questions/68738078/is-it-discouraged-to-import-conftest-py-within-test-modules